sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,158 @@ def __repr__(self):
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = (\mu_k f_n + (\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}) \text{sign}(v) + \sigma  v
+
+    where :math:`\mu_k` is the coefficient of kinetic friction, :math:`\mu_s` is the
+    coefficient of static friction, :math:`f_n` is the normal force, :math:`v` is the
+    relative velocity, :math:`v_s` is the Stribeck friction coefficient, and
+    :math:`\sigma` is the viscous friction constant.
+
+    The default friction force is :math:`F = \mu_k f_n`.
+    When specified, the actuator includes:
+
+    - Stribeck effect: :math:`(\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}`
+    - Viscous effect: :math:`\sigma v`
+
+    Notes
+    =====
+
+    The actuator makes the following assumptions:
+
+    - The actuator assumes relative motion is non-zero.
+    - The normal force is assumed to be a non-negative scalar.
+    - The resultant friction force is opposite to the velocity direction.
+    - Each point in the pathway is fixed within separate objects that are sliding relative to each other. In other words, these two points are fixed in the mutually sliding objects.
+
+    This actuator has been tested for straightforward motions, like a block sliding
+    on a surface.
+
+    The friction force is defined to always oppose the direction of relative velocity :math:`v`.
+    Specifically:
+
+    - The default Coulomb friction force :math:`\mu_k f_n \text{sign}(v)` is opposite to :math:`v`.
+    - The Stribeck effect :math:`(\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2} \text{sign}(v)` is also opposite to :math:`v`.
+    - The viscous friction term :math:`\sigma v` is opposite to :math:`v`.
+
+    Examples
+    ========
+
+    The below example shows how to generate the loads produced by a Coulomb kinetic
+    friction actuator in a mass-spring system with friction.
+
+    >>> import sympy as sm
+    >>> from sympy.physics.mechanics import (dynamicsymbols, ReferenceFrame, Point,
+    ...     LinearPathway, CoulombKineticFriction, LinearSpring, KanesMethod, Particle)
+
+    >>> x, v = dynamicsymbols('x, v', real=True)
+    >>> m, g, k, mu_k, mu_s, v_s, sigma = sm.symbols('m, g, k, mu_k, mu_s, v_s, sigma')
+
+    >>> N = ReferenceFrame('N')
+    >>> O, P = Point('O'), Point('P')
+    >>> O.set_vel(N, 0)
+    >>> P.set_pos(O, x*N.x)
+    >>> P.set_vel(N, v*N.x)
+
+    >>> pathway = LinearPathway(O, P)
+    >>> friction = CoulombKineticFriction(mu_k, m*g, pathway, v_s=v_s, sigma=sigma, mu_s=mu_k)
+    >>> spring = LinearSpring(k, pathway)
+    >>> block = Particle('block', point=P, mass=m)
+
+    >>> kane = KanesMethod(N, (x,), (v,), kd_eqs=(x.diff() - v,))
+    >>> friction.to_loads()
+        [(O, (g*m*mu_k*sign(sign(x(t))*Derivative(x(t), t)) + sigma*sign(x(t))*Derivative(x(t), t))*x(t)/Abs(x(t))*N.x), (P, (-g*m*mu_k*sign(sign(x(t))*Derivative(x(t), t)) - sigma*sign(x(t))*Derivative(x(t), t))*x(t)/Abs(x(t))*N.x)]
```

Comment:
This is what `to_loads()` gives for the force on P:

```
(P, (-g*m*mu_k*sign(sign(x(t))*Derivative(x(t), t)) - sigma*sign(x(t))*Derivative(x(t), t))*x(t)/Abs(x(t))*N.x)
```

written with math:

$$
\vec{F}^P = (-mg\mu_k \mathrm{sgn}(\mathrm{sgn}(x)\dot{x}) - \sigma\mathrm{sgn}(x)\dot{x}\frac{x}{|x|} \hat{n}_x
$$

Why isn't `v` present? And why is `x` present?

The formulation is also quite complex, but maybe that is required to guaranteed the signs are correct and have the most generality.

Reply:
I expected to have `v` as a velocity instead of `Derivative(x(t), t)` since I defined the velocity of P as `P.set_vel(N, v*N.x)` and the pathway as `pathway = LinearPathway(O, P)`, but I think we have `Derivative(x(t), t)` because `v` is internally resolved into its fundamental derivative form, `Derivative(x(t), t)`, which also allows symbolic consistency. `sign(x(t))` I think appears to correctly handle the direction of the friction force.

Or maybe I need further investigation on this.

Reply:
I'm recalling now that that is how pathways work. The pathway doesn't know what reference frame is the correct one to take the derivative in, so it assumes a reference frame in which the first point is fixed in and then calculates the time derivative of the position vector to the second point. This means it cannot make use of any velocity that is predefined in any frame because the pathway is unaware what frames the two points might be fixed in.

Reply:
So, I think we are stuck with not having `v` present.

Reply:
The extension velocity is defined as the length change over time, i.e. Derivative(length, t),  and the direction of the force is simply the relative position divided by the distance between the points. This introduces some ugly terms, but there isn't really a way around it.

Reply:
I think the problem with the limitation of using predefined velocity and the desired reference frame lies in the pathway rather than the actuator. 

So, we can explain in the docstring that users can create their own custom pathway, similar to the `ExtensorPathway` in the `biomechanical-model-example` docs with length, extension_velocity, and to_loads properties, and present example using a custom pathway in either the docstring or the separate docs.

Reply:
I don't think we need to do anything here, it is just the way pathways work and should be explained in the pathway docs.